### PR TITLE
CircleCI Windows build

### PR DESCRIPTION
**Pull request by mingbowan**
use beefier GPU machine for build and small ones for test

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -19,6 +19,21 @@ docker_config_defaults: &docker_config_defaults
...
     aws_secret_access_key: ${CIRCLECI_AWS_SECRET_KEY_FOR_ECR_READ_WRITE_V4}
+
+executors:
+  windows-gpu:
+    machine:
+      resource_class: windows.gpu.small.prototype
+      image: windows-server-2019-nvidia:edge
+      shell: bash.exe
+
+  windows-cuda:
```
Comment (suo) [inline .circleci/config.yml]: windows-gpu and windows-cuda are somewhat ambiguous names I think. Why not `windows-gpu` and `windows-cpu`/`windows-cpu-with-cuda` or whatever?

**Review by suo — APPROVED** [review]
great :) let's try running it for a few days before disabling jenkins

File: .circleci/config.yml (hunk 1 above)
Comment (mingbowan) [reply .circleci/config.yml]: Sounds good

**Comment by mingbowan** [comment]
replaced by https://github.com/pytorch/pytorch/pull/30556

**Pull request closed without merging**